real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/RBSMNode.java

```diff
@@ -144,4 +145,35 @@ public int peel(byte[] buffer, int start, int bytes_available) {
 		return 6;
 	}
 
+	
+	@SuppressWarnings("unchecked")
+	public static JSONObject translatePeelMessageToJObject(String message) {
+		String[] messageData = message.split(",");
+		String sensorName = messageData[0];
+		sensorName = sensorName.substring(sensorName.indexOf("/") + 1);
+		JSONObject data = new JSONObject();
+		JSONObject params = new JSONObject();
+		
+		data.put("timestamp", messageData[1]);
+		
+		if(sensorName.equals("steering")) {
+			data.put("name", "Steering");
+			params.put("angle", Float.valueOf(messageData[2]));
+		}
+		else if (sensorName.equals("encoder")) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1278054652/issues/source?bid=2938665&fileBranchId=2938743#l163)